### ISSUE TITLE
the gradient of bias in jax.nn.dot_product_attention

### ISSUE THREAD

**Issue by bojone** [jax-ml/jax#27184]
### Description

I'd like to confirm whether it is normal that the `bias` parameter has no gradient when `implement='cudnn'` in `jax.nn.dot_product_attention`.

### System info (python version, jaxlib version, accelerator, etc.)

jax 0.5.0
